code/game/machinery/vendor/weapon_kit_vendors.dm

```diff
@@ -239,6 +241,7 @@
 /obj/machinery/vending/marshal_kit/proc/RedeemSpecPrimary(obj/item/voucher/voucher, mob/redeemer)
 	var/items = list(
 					"Thompson Kit" = /obj/item/storage/box/m_kit/thompson,
+					"Breacher-hammer Kit" = /obj/item/storage/box/m_kit/breacher,
```

Comment:
Why is this spec only?

Reply:
because i put it in the wrong voucher :)